docs/monoDrive_home/Vehicle-Configuration.md

```diff
@@ -1,89 +1,208 @@
-# Vehicles
+# Scenario Vehicle
+The monoDrive Simulator allows you to customize multiple visual and dynamic settings on the ego vehicle.
+
```

Comment:
Try not using the second person in the docs. Also, what is the difference between visual and dynamic settings? Just to make this intro look fuller, this could be a place to explore.  The monoDrive Simulator provides users with the ability to customize multiple visual and dynamic settings on the ego vehicle.

Reply:
Is this better?

Reply:
This might be better: The monoDrive Simulator provides users with the ability to customize color and the model for the ego vehicle as well as for the vehicles that interact with it on simulation. The user may specify the initial conditions for any car on the scene, i.e. initial pose and initial velocity.